index.html

```diff
@@ -0,0 +1,76 @@
+        </svg>
+        <svg width="16" height="12" viewBox="0 0 16 12" fill="none" xmlns="http://www.w3.org/2000/svg" class="phone">
+          <path fill-rule="evenodd" clip-rule="evenodd" d="M8.33045 2.60838C10.5463 2.60848 12.6775 3.45992 14.2835 4.98672C14.4044 5.10459 14.5977 5.1031 14.7168 4.98338L15.8728 3.81672C15.9331 3.75599 15.9667 3.67374 15.9662 3.58816C15.9657 3.50258 15.9311 3.42073 15.8701 3.36072C11.655 -0.678832 5.00522 -0.678832 0.79012 3.36072C0.729062 3.42068 0.694418 3.50251 0.693855 3.58809C0.693292 3.67367 0.726856 3.75595 0.78712 3.81672L1.94345 4.98338C2.06248 5.10328 2.25593 5.10477 2.37679 4.98672C3.98294 3.45982 6.11434 2.60838 8.33045 2.60838ZM8.33045 6.40405C9.54794 6.40397 10.722 6.85651 11.6245 7.67372C11.7465 7.7897 11.9388 7.78718 12.0578 7.66805L13.2125 6.50138C13.2733 6.44019 13.307 6.35717 13.3061 6.27091C13.3052 6.18464 13.2698 6.10233 13.2078 6.04238C10.4596 3.486 6.20365 3.486 3.45545 6.04238C3.39338 6.10233 3.35796 6.18468 3.35714 6.27097C3.35633 6.35727 3.39019 6.44028 3.45112 6.50138L4.60545 7.66805C4.72444 7.78718 4.91672 7.7897 5.03879 7.67372C5.94066 6.85705 7.11377 6.40455 8.33045 6.40405ZM10.6434 8.95787C10.6452 9.04437 10.6112 9.12778 10.5495 9.18838L8.55212 11.204C8.49357 11.2633 8.41374 11.2966 8.33045 11.2966C8.24716 11.2966 8.16734 11.2633 8.10879 11.204L6.11112 9.18838C6.04941 9.12773 6.01546 9.0443 6.01729 8.9578C6.01912 8.87129 6.05657 8.78937 6.12079 8.73138C7.39636 7.6525 9.26455 7.6525 10.5401 8.73138C10.6043 8.78942 10.6417 8.87136 10.6434 8.95787Z" fill="black"/>
+        </svg>
+        <svg width="25" height="12" viewBox="0 0 25 12" fill="none" xmlns="http://www.w3.org/2000/svg" class="phone">
+          <rect opacity="0.35" x="0.5" y="0.833984" width="21" height="10.3333" rx="2.16667" stroke="black"/>
+          <path opacity="0.4" d="M23 4.00098V8.00098C23.8047 7.6622 24.328 6.87411 24.328 6.00098C24.328 5.12784 23.8047 4.33975 23 4.00098Z" fill="black"/>
+          <rect x="2" y="2.33398" width="18" height="7.33333" rx="1.33333" fill="black" />
+        </svg>
+      </div>
+    <div class="header2">
+      <div class="logo_welcome">
+        <p class="wlc">Welcome</p>
+      </div>
```

Comment:
- [ ] The Header should be aligned with flexbox, This gives us a lot of options to align all our elements, plus, it's responsive.
- [ ] And kindly fix the CSS for your header to match the Figma design: make sure the logo and the navbar are aligned in the center and I would love to suggest to justify them using justify-between

![image](https://user-images.githubusercontent.com/89282221/212153188-aafd6d05-0d7b-4cc8-b23b-cd5dfd68bd3c.png)